website/blog/2022-01-20-initial-setup.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Initial setup and first steps
+description: This blog post describes how to set up Eightshift Boilerplate theme, as well as covers some most common issues that may occur
+slug: initial-setup
+authors: iobrado
+tags: [eightshift, boilerplate, setup]
+image: #
+hide_table_of_contents: false
+---
+As it is quite often the case when trying out something new, it can be quite likely that you run into some hiccups. You're trying out our Eightshift Boilerplate, but simply can't get it to work? Worry not, we'll go through the setup and the most common issues that may occur.
+
+<!--truncate-->
+If you haven't already, be sure to read our official [Eightshift Docs](https://infinum.github.io/eightshift-docs/docs/welcome).
+
+### Setup
+
+As it is already mentioned in documentation, to create Eightshift theme, navigate to `wp-content/themes` in your project and type the following in the terminal:
+
+`npx create-wp-project`
+
+OK, you went through the process, filled out all required information, but the theme still isn't working. There are few more instructions in terminal what to do next, but you may have skipped them. While the principle **"6 hours of debugging can save you 5 minutes of reading documentation"** is fun, sometimes it's just not worth it. Here's what you should do:
+
+- run `wp theme activate your-theme-name` to activate your new theme
+- next, navigate to `wp-content/themes/your-theme-name`
+- try running `wp boilerplate --help` to check if our WP CLI boilerplate commands work
+- for easiest initial setup, run `wp boilerplate setup_theme`
+- finally, run `npm start`
```

Comment:
`setup_theme` will run the npm for you after the initial setup 😉 

https://github.com/infinum/eightshift-libs/blob/main/src/Cli/CliInitTheme.php#L103

Reply:
Yeah, it does, I just re-tested the whole process again. Still, I would like to leave that in some form. Maybe something like:
```suggestion
- `npm start` should trigger automatically with the previous command. If it doesn't, you can run it manually to finish the theme build process
```

Reply:
Jup, sounds good to me 👍🏼